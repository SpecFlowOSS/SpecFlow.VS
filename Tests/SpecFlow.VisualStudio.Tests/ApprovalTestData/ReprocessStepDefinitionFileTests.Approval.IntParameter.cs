using System;
using TechTalk.SpecFlow;

namespace TestProject
{
    [Binding]
    public class Feature1StepDefinitions
    {
        [When(@"I press (.*)")]
        public void WhenIPress(int intParameter)
        {
            throw new PendingStepException();
        }
    }
}
