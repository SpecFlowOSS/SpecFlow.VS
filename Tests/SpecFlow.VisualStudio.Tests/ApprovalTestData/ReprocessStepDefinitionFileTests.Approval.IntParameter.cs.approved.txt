ProjectBindingRegistry:
  ProjectStepDefinitionBinding-0:
    IsValid:`True`
    Error:``
    StepDefinitionType:`When`
    SpecifiedExpression:`I press (.*)`
    Regex:`^I press (.*)$`
    Scope:``
    Implementation:
      Method:`TestProject.Feature1StepDefinitions.WhenIPress`
      ParameterTypes:[1]
        - `int`
      SourceLocation:
        SourceFile:`C:\<Full path to>\IntParameter.cs`
        SourceFileColumn:`9`
        SourceFileEndColumn:`9`
        SourceFileEndLine:`13`
        SourceLocationSpan:``
        SourceFileLine:`11`
    Expression:`I press (.*)`
